### PR TITLE
[ADD] Move hr_contract_signature in acsone-addons

### PR DESCRIPTION
**Pull request by adrienpeiffer**
cc\ @sbidoul

### CODE REVIEW

File: hr_contract_signature/pre_install.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+def rename_module(cr):
+    old_names = ['acsone_hr_contract']
+    new_name = 'hr_contract_signature'
+    query = ("UPDATE ir_model_data SET module = %s "
+             "WHERE module in %s ")
+    logged_query(cr, query, (new_name, tuple(old_names)))
+    query = ("UPDATE ir_module_module_dependency SET name = %s "
+             "WHERE name in %s")
+    logged_query(cr, query, (new_name, tuple(old_names)))
```
Comment (sbidoul) [inline hr_contract_signature/pre_install.py:47]: Is it not possible to use openupgradelib for this rename operations?

Comment (adrienpeiffer) [inline hr_contract_signature/pre_install.py:47]: I tried to use the rename method in openupgradelib. But it'doesn't work really. I can investigate in this way.

File: hr_contract_signature/models/hr_contract.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+##############################################################################
+
+from openerp import models, fields
+
+
+class hr_contract(models.Model):
+    _inherit = "hr.contract"
+
+    contract_signed = fields.Boolean(
+        string='Contract Signed',
```
Comment (JonathanNEMRY) [inline hr_contract_signature/models/hr_contract.py]: string attribute is not required in this case (https://www.odoo.com/documentation/8.0/reference/orm.html#creating-models)

**Comment by adrienpeiffer** [comment]
Thanks @JonathanNEMRY !

**Pull request merged**